dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
I don't think there is a lot of overlap, each has a different focus:

* `fabric` focuses on deployment and host installs
* `docker-compose` focuses on isolated development environments
* `dobi` focuses on portable, and automated build operations

There is some overlap in fabric and dobi right now, but fabric requires that you're running an SSH server and you want to install all the dependencies on your host. This install can fail pretty easily if you're running an unexpected version of ubuntu, or will definitely fail for any non-ubuntu distro (I run arch).

With the `dobi` + `docker` setup you can be fairly confident that the install is going to work on any platform (linux, osx, or windows). It also simplifies the install instructions. Instead of pages of readme, you can summarize the entire process in two lines (as shown in the OP). This could even be reduced to a single line if we create a task for "create admin user".

The "deployment" and "admin" tasks from `fabric` can still be used from the prompt created by `dobi shell`